pwndbg/aglib/memory.py

```diff
@@ -338,22 +338,62 @@ def find_lower_boundary(addr: int, max_pages: int = 1024) -> int:
 
 
 def update_min_addr() -> None:
-    # FIXME: Debugger-agnostic API can't do QEMU detection yet.
-    #
-    # global MMAP_MIN_ADDR
-    # if pwndbg.gdblib.qemu.is_qemu_kernel():
-    #    MMAP_MIN_ADDR = 0
-    pass
+    global MMAP_MIN_ADDR
+    if pwndbg.aglib.qemu.is_qemu_kernel():
+        MMAP_MIN_ADDR = 0
```

Comment:
shouldnt there be some else branch? what if we're not in qemu kernel, shouldnt we bring back old value or sth?

Reply:
Can u open another PR fixing this? since this seems like a bug. Should be:

`MMAP_MIN_ADDR = 0 if if pwndbg.aglib.qemu.is_qemu_kernel() else 0x8000`